src/components/BooksPage.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { ListGroup, Button, Form } from 'react-bootstrap';
+import './BookPageStyle.css';
+
+const BooksPage = () => (
+  <main>
+    <section className="books-page">
+      <h1>Books List</h1>
+      <ListGroup>
+        <ListGroup.Item>
+          Cras justo odio
+          <Button variant="secondary">Remove</Button>
+        </ListGroup.Item>
+        <ListGroup.Item>
+          Dapibus ac facilisis in
+          <Button variant="secondary">Remove</Button>
+        </ListGroup.Item>
+        <ListGroup.Item>
+          Morbi leo risus
+          <Button variant="secondary">Remove</Button>
+        </ListGroup.Item>
+        <ListGroup.Item>
+          Porta ac consectetur ac
+          <Button variant="secondary">Remove</Button>
+        </ListGroup.Item>
+        <ListGroup.Item>
+          Vestibulum at eros
+          <Button variant="secondary">Remove</Button>
+        </ListGroup.Item>
```

Comment:
- [ ] The data of your books should be passed as a `prop`, like this:
    - `<Book title={books[0]} />`, I would recommend you to have a list of books on this component, in order to pass data to each `<Book/>` component 😄

Reply:
Thanks, @helman101  for the review! 😄